batchflow/models/tf/base.py

```diff
@@ -1056,7 +1071,30 @@ def save(self, path, *args, **kwargs):
                 self._saver = tf.train.Saver()
             self._saver.save(self.session, os.path.join(path, 'model'), *args, global_step=self.global_step, **kwargs)
             with open(os.path.join(path, 'attrs.json'), 'w') as f:
-                json.dump(self._attrs, f)
+                attrs_path = dict()
+                for attr in self._attrs:
+                    attr_additional = getattr(self, attr)
+                    if isinstance(attr_additional, dict):
+                        attrs_path.update({attr: self._conversion_to_name(attr_additional.copy())})
+                    else:
+                        attrs_path.update({attr: self._conversion_to_name(attr_additional)})
+                json.dump(attrs_path, f)
+
+    def _conversion_to_graph_item(self, name):
```

Comment:
```suggestion
    def _to_graph_item(self, name):
```
`to` means `conversion` on its own